kiwiirc/kiwiirc
# Allow closing all queries at once, remove unneeded close buttons from container header

**How to test:**
open 2 queries to any users, the close button will appear on the "Messages" group

**The issue:**
the animated "Really close all queries?" box does not float in behind statebrowser

note: it currently does not come all the way out, this is intentional to assist debugging the z-index issues
